composer.json

```
@@ -9,21 +9,21 @@
   "homepage": "https://github.com/webonyx/graphql-php",
   "license": "MIT",
   "require": {
-    "php": "^7.4 || ^8.0",
+    "php": "^7.4 || ^8",
     "ext-json": "*",
     "ext-mbstring": "*"
   },
   "require-dev": {
-    "amphp/amp": "^2.3",
-    "dms/phpunit-arraysubset-asserts": "^0.2.1",
-    "doctrine/coding-standard": "^9.0",
-    "ergebnis/composer-normalize": "^2.13",
-    "nyholm/psr7": "^1.2",
-    "phpbench/phpbench": "^1",
-    "phpstan/extension-installer": "^1",
-    "phpstan/phpstan": "0.12.99",
-    "phpstan/phpstan-phpunit": "0.12.22",
-    "phpstan/phpstan-strict-rules": "0.12.11",
+    "amphp/amp": "^2.6",
+    "dms/phpunit-arraysubset-asserts": "^0.3",
+    "doctrine/coding-standard": "^9",
+    "ergebnis/composer-normalize": "^2.16",
+    "nyholm/psr7": "^1.4",
+    "phpbench/phpbench": "^1.2",
+    "phpstan/extension-installer": "^1.1",
+    "phpstan/phpstan": "1.2.0",
```

for v1.x.y we can use 

```suggestion
    "phpstan/phpstan": "^1.2.0",
```

---

Let's try it. We might have to revert to a fixed version regardless of the 1.x status, because PHPStan might find new errors in newer versions which would then randomly fail CI in completely unrelated PRs.